pom.xml

```diff
@@ -419,6 +420,24 @@
           </rules>
         </configuration>
       </plugin>
+
+      <plugin>
```

Comment:
I'll pull this up to the mybatis parent before the next release at least in general configuration so other projects can benefit from it as many of the projects already have some of the exact same formatting enabled.  I think this plugin can possibly use the hook from a jar and will look into that just in case that is possible so we don't need to repeat any of it.  For now this is fine.